integration-tests/contracts/contract_deployer.go

```diff
@@ -763,6 +764,25 @@ func (e *EthereumContractDeployer) DeployUpkeepTranscoder() (UpkeepTranscoder, e
 	}, err
 }
 
+func (e *EthereumContractDeployer) LoadUpkeepTranscoder(address common.Address) (UpkeepTranscoder, error) {
+	instance, err := e.client.LoadContract("UpkeepTranscoder", address, func(
+		address common.Address,
+		backend bind.ContractBackend,
+	) (interface{}, error) {
+		return upkeep_transcoder.NewUpkeepTranscoder(address, backend)
+	})
+
```

Comment:
🏄‍♂️ I would drop this new line

Reply:
Have another task to update the upkeep transcoder version. Will clean up then.